enterprise/reporting/flusher/flusher.go

```diff
@@ -303,6 +315,23 @@ func (f *Flusher) delete(ctx context.Context, minReportedAt, maxReportedAt time.
 	return err
 }
 
+func (f *Flusher) vacuum(ctx context.Context) error {
+	vacuumStart := time.Now()
+	var sizeEstimate int64
+	if err := f.db.QueryRowContext(
+		ctx, fmt.Sprintf(`SELECT pg_table_size(oid) from pg_class where relname='%s'`, f.table),
+	).Scan(&sizeEstimate); err != nil {
+		return fmt.Errorf("error getting table size %w", err)
+	}
+	if sizeEstimate > f.vacuumThresholdBytes.Load() {
+		if _, err := f.db.ExecContext(ctx, fmt.Sprintf("vacuum full analyze %s", f.table)); err != nil {
```

Comment:
We can use QuoteIdentifier, to be safely quote the table name.
```suggestion
		if _, err := f.db.ExecContext(ctx, fmt.Sprintf("vacuum full analyze %s", pq.QuoteIdentifier(f.table))); err != nil {
```

Reply:
we don't need to quote table name here

Reply:
Yeah we don't essentially need it but it safer approach to passing the table name via QuoteIdentifier.